easybuild/easyconfigs/p/Python/Python-2.7.17-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,293 @@
+    }),
+    ('py_expression_eval', '0.3.9', {
+        'checksums': ['d80a948f91f78d08f789b0a7c3fb2bd9a34ad625f5ce88c262a6c91189a4abb9'],
+    }),
+    ('tabulate', '0.8.6', {
+        'checksums': ['5470cc6687a091c7042cee89b2946d9235fe9f6d49c193a4ae2ac7bf386737c8'],
+    }),
+    ('ujson', '1.35', {
+        'checksums': ['f66073e5506e91d204ab0c614a148d5aa938bdbf104751be66f8ad7a222f5f86'],
+    }),
+    ('atomicwrites', '1.3.0', {
+        'checksums': ['75a9445bac02d8d058d5e1fe689654ba5a6556a1dfd8ce6ec55a0ed79866cfa6'],
+    }),
+    ('py', '1.8.1', {
+        'checksums': ['5e27081401262157467ad6e7f851b7aa402c5852dbcb3dae06768434de5752aa'],
+    }),
+    ('scandir', '1.10.0', {
+        'checksums': ['4d4631f6062e658e9007ab3149a9b914f3548cb38bfb021c64f39a025ce578ae'],
+    }),
+    ('pathlib2', '2.3.5', {
+        'checksums': ['6cd9a47b597b37cc57de1c05e56fb1a1c9cc9fab04fe78c29acd090418529868'],
+    }),
+    ('zipp', '1.2.0', {
+        'checksums': ['c70410551488251b0fee67b460fb9a536af8d6f9f008ad10ac51f615b6a521b1'],
+    }),
+    ('configparser', '4.0.2', {
+        'checksums': ['c7d282687a5308319bf3d2e7706e575c635b0a470342641c93bea0ea3b5331df'],
+    }),
+    ('contextlib2', '0.6.0.post1', {
+        'checksums': ['01f490098c18b19d2bd5bb5dc445b2054d2fa97f09a4280ba2c5f3c394c8162e'],
+    }),
+    ('importlib_metadata', '1.5.0', {
+        'checksums': ['06f5b3a99029c7134207dd882428a66992a9de2bef7c2b699b5641f9886c3302'],
+    }),
+    ('pluggy', '0.13.1', {
+        'checksums': ['15b2acde666561e1298d71b523007ed7364de07029219b604cf808bfa1c765b0'],
+    }),
+    # more-itertools 5.0.0 is the most recent version still supporting Python 2.x
+    ('more-itertools', '5.0.0', {
+        'checksums': ['38a936c0a6d98a38bcc2d03fdaaedaba9f412879461dd2ceff8d37564d6522e4'],
+    }),
+    ('attrs', '19.3.0', {
+        'modulename': 'attr',
+        'checksums': ['f7b7ce16570fe9965acd6d30101a28f62fb4a7f9e926b3bbc9b61f8b04247e72'],
+    }),
+    ('wcwidth', '0.1.8', {
+        'checksums': ['f28b3e8a6483e5d49e7f8949ac1a78314e740333ae305b4ba5defd3e74fb37a8'],
+    }),
+    # pytest 4.6.x is the most recent version still supporting Python 2.x
+    ('pytest', '4.6.9', {
+        'checksums': ['19e8f75eac01dd3f211edd465b39efbcbdc8fc5f7866d7dd49fedb30d8adf339'],
+    }),
+    ('MarkupSafe', '1.1.1', {
+        'checksums': ['29872e92839765e546828bb7754a68c418d927cd064fd4708fab9fe9c8bb116b'],
+    }),
+    ('Jinja2', '2.11.1', {
+        'checksums': ['93187ffbc7808079673ef52771baa950426fd664d3aad1d0fa3e95644360e250'],
+    }),
+    ('packaging', '20.1', {
+        'checksums': ['e665345f9eef0c621aa0bf2f8d78cf6d21904eef16a93f020240b704a57f1334'],
+    }),
+    # sphinxcontrib-websupport 1.1.2 is the most recent version still supporting Python 2.x
+    ('sphinxcontrib-websupport', '1.1.2', {
+        'modulename': 'sphinxcontrib.websupport',
+        'checksums': ['1501befb0fdf1d1c29a800fdbf4ef5dc5369377300ddbdd16d2cd40e54c6eefc'],
```

Comment:
The other sphinxcontrib extensions from 3.8.2 are missing, why?

Reply:
All of the other sphinxcontrib extensions are requirements with Sphinx >= 2 and require Python >= 3.5 (in all commits of the code). I think these were split out from Sphinx in the jump in versions from 1 to 2.